libsvmdata/datasets.py

```diff
@@ -33,19 +35,24 @@
     'rcv1_train': 'binary/rcv1_train.binary.bz2',
     'real-sim': 'binary/real-sim.bz2',
     'sector_train': 'multiclass/sector/sector.bz2',
+    'sector_scale': 'multiclass/sector/sector.scale.bz2',
```

Comment:
can you keep the dict keys them in alphabetical order ? (so move it one line up)